packages/common/src/db/models.ts

```diff
@@ -223,6 +223,10 @@ export const OpenGovDelegationModel = mongoose.model(
 OpenGovDelegationSchema.index({ delegate: 1, track: 1 });
 
 export const CandidateSchema = new Schema({
+  // The unique identifier of the candidate's node for a given slot.
+  slotId: Number,
```

Comment:
what about ?
slotId: { type: Number, unique: true, required: true, index: true }

Reply:
the index:true field should be important, since you probably want to query by slotId elsewhere

Reply:
Yes, agree - will make it indexed and unique 👍